apps/notifications/src/panel/templates/body.jsx

```diff
@@ -114,6 +115,31 @@ export class NoteBody extends React.Component {
 					);
 					break;
 				case 'post':
+					slideCount = 0;
+					for ( i = 0; i < block.block.ranges.length; i++ ) {
+						if ( block.block.ranges[ i ].type === 'figcaption' ) {
```

Comment:
Perhaps we can assign `block.block.ranges[ i ]` to a variable.

Reply:
Thank you for suggestion Yan! I am now addressing that in the following commit: https://github.com/Automattic/wp-calypso/pull/55062/commits/3e456bd3f42de201bff1f5dbbe5e65babad2db63.